### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by MoRaad97**
###In this PR We:
- Add the HTML structure of the menu section.
- Add style using pure CSS.
- Add the js Dom event to the menu.
- Add Linters js file.

### CODE REVIEW

File: css/master.css [hunk 1]
```diff
@@ -196,7 +257,7 @@
 
   100% {
     transform: rotateZ(360deg);
-    transition-duration: 1;
+    transition-duration: 1s;
   }
 }
```
Comment (BenMukebo) [inline css/master.css:274]: - [x] Great job so far! 👏🏻  however according to the project requirement, on the desktop size, the hamburger menu button, the closed button, and the menu page are not visible. Kindly correct this. You can use your @media (min-width: 768px) to display none of your navigation menu.

File: css/master.css [hunk 2]
```diff
@@ -6,6 +6,14 @@
   --section-main-padding: 50px;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
```
Comment (BenMukebo) [inline css/master.css:12]: - [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository have been set up correctly. ✔️  
- Good commit history ✔️ 
-  When the user clicks the hamburger button, the mobile menu appears.
- When the user clicks the close (X) button, the mobile menu disappears.
- When the user clicks any of the mobile menu options, a correct part is displayed and the mobile menu disappears 👏🏻 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @MoRaad97, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**